### ISSUE TITLE
tmux quit siently

### ISSUE THREAD

**Issue by machsix** [tmux/tmux#3153]
### Issue description

tmux quits silently after a recent update of the cluster.

### Required information

Please provide the following information:

* tmux version (`tmux -V`). tmux 3.2a
* Platform (`uname -sp`). Linux x86_64
* $TERM inside and outside of tmux (`echo $TERM`). xterm-256color
* Logs from tmux (`tmux kill-server; tmux -vv new`).
[tmux-server-18524.log](https://github.com/tmux/tmux/files/8475417/tmux-server-18524.log)

**Comment by nicm** [comment]
Something on your system is broken:
~~~~
1649786591.307636 cmdq_error: create window failed: fork failed: Operation not permitted
~~~~
My guess is your pty devices are wrong in some way.

**Comment by machsix** [comment]
Hi nicm, the HPC support team changed the old version tmux on the system to the following and it works
```
-rwsr-xr-x 1 root root 423120 Jan 27  2014 /usr/bin/tmux  
```

But my self-built one still complains about the same error. Any idea where can I debug? Thank you!

**Comment by nicm** [comment]
Do this `strace -vvttfostrace.out tmux new` and show me the strace.out file.

**Comment by nicm** [comment]
Yes, it looks like your /dev/pts has the wrong permissions or is otherwise misconfigured:
~~~~
120068 08:41:55.750827 chown("/dev/pts/70", 30154, 5) = -1 EPERM (Operation not permitted)
~~~~
Are you sure devpts is mounted with the correct gid and mode?

**Comment by machsix** [comment]
Thanks nicm! The /dev/pts was indeed not mounted correctly.  The file /etc/mtab shows
```
devpts /dev/pts devpts rw,nosuid,noexec,relatime,mode=600,ptmxmode=000 0 0
```
It should be like 
```
devpts /dev/pts devpts rw,nosuid,noexec,noatime,gid=5,mode=620,ptmxmode=000 0 0 
```
http://web.archive.org/web/20220407165609/https://bugzilla.redhat.com/show_bug.cgi?id=884673#c7 also has an explanation
